<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aaaaa</class>
 <widget class="QDialog" name="aaaaa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background:url(C:/hammerrrm.jpg) ;
}</string>
  </property>
  <widget class="QFrame" name="add_employee">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>-30</y>
     <width>501</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background:#51556e;
    border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/QTprojet/smart court.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="MODIFIER">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:white;
border-radius: 15px;
background:#303030;
border: 2px solid #bfbeba;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white;
border: 2px solid #bfbeba;
}</string>
    </property>
    <property name="text">
     <string>UPDATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_num">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>etat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_etat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
